openapi/components/schemas/Config.yaml

```diff
@@ -200,6 +200,7 @@ properties:
           minLength: 1
         tag:
           type: string
+          nullable: true
```

Comment:
Tag can never be `null`, although it does seem like it is sometimes left out. Please remove it from the `required` list instead.

Reply:
Done